presets/4.3/tune/mouseFPV_AOS_35_tune_filters.txt

```diff
@@ -0,0 +1,168 @@
+#$ TITLE:  AOS 3.5 V1 Tune | mouseFPV
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: AOS, 4s
+#$ AUTHOR: mouseFPV
+#$ DESCRIPTION: Tune for AOS 3.5 with FPVCycle 16mm motors, <250g 4s
+#$ DESCRIPTION: May be approproiate for v2 frame and/or different motors
+#$ DESCRIPTION: Tune/Filters are aggressive, and may not be approproiate for non trussed/braced frames
+#$ DESCRIPTION: Recommended 48k PWM
+#$ DESCRIPTION: If you use the filters and are on an F4, use a 4k PID Loop
+#$ WARNING: Included Filters set motor poles to 12! Filters Require RPM filtering and DShot telemetry!
+#$ DISCLAIMER: Flash At Your Own Risk!
+#$ FORCE_OPTIONS_REVIEW: TRUE
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+# ------- End Defaults --------
+# ----- Begin Mouse Tune-------
+
+# -- PID Sliders  --
+set simplified_pids_mode = RPY
+set simplified_d_gain = 130
+set simplified_pi_gain = 100
+set simplified_feedforward_gain = 60
+set simplified_dmax_gain = 000
+set simplified_i_gain = 80
+set simplified_pitch_d_gain = 105
+set simplified_pitch_pi_gain = 105
+set simplified_master_multiplier = 145
+simplified_tuning apply
+
+# -- iTerm relax --
+set iterm_relax = RP
+set iterm_relax_type = SETPOINT
+set iterm_relax_cutoff = 10
+
+# -- TPA  --
+set tpa_rate = 70
+
+# -- Feedforward --
+set feedforward_boost = 15
+
+# -- Feedforward Jitter Reduction --
+set feedforward_jitter_factor = 14
+
+# -- Antigravity --
+set anti_gravity_gain = 3500
+
+# -- Thrust linear --
+set thrust_linear = 20
+
+# -- DShot Idle (default)--
+# Commonly set lower when dynamic idle is active.
+set dshot_idle_value = 400
+
+# -- Dyn Idle (off, default) --
+set dyn_idle_min_rpm = 0
+
+# -- VBat sag compensation --
+set vbat_sag_compensation = 100
+
+
+# ------ OPTIONS GO BELOW THIS LINE ------
+
+
+# This is where the author includes options that require input from the User
+
+#$ OPTION BEGIN (CHECKED): FPVCycle 16mm Motors?
+set motor_output_limit = 95
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Dynamic Idle
+set dshot_bidir = ON
+set dshot_idle_value = 150
+set dyn_idle_min_rpm = 25
+set dyn_idle_p_gain = 50
+#$ OPTION END
+
+#$ OPTION_GROUP BEGIN: Tunes (Choose One or None)
+
+#$ OPTION BEGIN (Unchecked): Include RPM Filter (F7)?
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+# ------- End Defaults --------
+# ----- Begin Mouse Tune-------
+
+# enable dshot rpm telemetry
+set dshot_bidir = ON
+set motor_poles = 12
+
+# -- Gyro lowpass filters --
+# -- No Gyro Lowpass
+set gyro_lpf1_static_hz = 0
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set gyro_lpf2_static_hz = 0
+set simplified_gyro_filter_multiplier = 200
```

Comment:
whats the purpose of
set simplified_gyro_filter_multiplier = 200
when you turn it off later?